app/presenters/menu/default_menu.rb

```diff
@@ -186,6 +186,13 @@ def automate_menu_section
           Menu::Item.new('miq_ae_class',         N_('Explorer'),        'miq_ae_class_explorer',         {:feature => 'miq_ae_domain_view'},            '/miq_ae_class/explorer'),
           Menu::Item.new('miq_ae_tools',         N_('Simulation'),      'miq_ae_class_simulation',       {:feature => 'miq_ae_class_simulation'},       '/miq_ae_tools/resolve'),
           Menu::Item.new('miq_ae_customization', N_('Customization'),   'miq_ae_customization_explorer', {:feature => 'miq_ae_customization_explorer'}, '/miq_ae_customization/explorer'),
+          Menu::Item.new(
+            'generic_object_definition',
+            N_('Generic Object Definitions'),
+            'generic_object_definition_explorer',
+            {:feature => 'generic_object_definition_explorer'},
+            '/generic_object_definition/explorer'
+          ),
```

Comment:
I had it aligned, but it stretched really far cause of the long names, that still ok?